samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -637,6 +637,20 @@ static void radio_disable(void)
 #endif /* CONFIG_FEM */
 }
 
+#if defined(NRF54L_SERIES)
+void mltpan_6(nrf_radio_mode_t mode)
+{
+	if (mode == NRF_RADIO_MODE_IEEE802154_250KBIT) {
+		*((volatile uint32_t *)0x5008A810) = 2;
+	}
+}
+#else
+void mltpan_6(nrf_radio_mode_t mode)
+{
+	ARG_UNUSED(mode);
+}
+#endif /* defined(NRF54L_SERIES) */
+
```

Comment:
1. The function could be `static`, because only current translation unit will use this function.
2. The `#if defined...` could be inside, there is no need to repeat the parameters of the function.
```suggestion
static void mltpan_6(nrf_radio_mode_t mode)
{
#if defined(NRF54L_SERIES)
	if (mode == NRF_RADIO_MODE_IEEE802154_250KBIT) {
		*((volatile uint32_t *)0x5008A810) = 2;
	}
#else
	ARG_UNUSED(mode);
#endif /* defined(NRF54L_SERIES) */
}

```